app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,24 @@
+class RecipesController < ApplicationController
+  before_action :authenticate_user!
+  def index
+    @user = current_user
+    @recipes = @user.recipes
+  end
+
+  def show
+    @user = current_user
+    @recipe = Recipe.find(params[:id])
+    @recipe_foods = @recipe.recipe_foods.includes([:food])
+  end
+
+  def destroy
+    @recipe = Recipe.find(params[:id])
+    @recipe.destroy
+    respond_to do |format|
+      format.html do
+        redirect_to user_recipes_path(params[:user_id]),
+                    notice: 'recipe deleted successfully'
+      end
```

Comment:
- [ ] It will be great to create success and fail paths for your destroy action. If the resource is deleted, do something. If it is not deleted do something else. This will prevent your application from misbehaving incase something unexpected happens

Reply:
No, this is not optional, when you implement either create destroy or update in your application, is a requirement for us to check if you have done it right. And the only way to do it right is to ensure you have created both success and fail paths